lib/testing/src/lib/core/pages/form/widgets/attach-file-widget.page.ts

```diff
@@ -36,18 +36,36 @@ export class AttachFileWidgetPage {
     async attachFile(fieldId, fileLocation): Promise<void> {
         browser.setFileDetector(new remote.FileDetector());
         const widget = await this.formFields.getWidget(fieldId);
-        const uploadButton = await widget.element(this.uploadLocator);
+        const uploadButton = await widget.element(this.alfrescoTypeUploadLocator);
         await BrowserActions.click(uploadButton);
         await BrowserVisibility.waitUntilElementIsPresent(this.localStorageButton);
         await this.localStorageButton.sendKeys(fileLocation);
     }
 
+    async checkNoFileIsAttached(fieldId): Promise<void> {
```

Comment:
Not sure I can really understand the difference from the name of:
checkNoFileIsAttached checkFileIsNotAttached
checkFileIsAttached checkFilesAreAttachedToWidget

Reply:
checkNoFileIsAttached -> this checks that no file is attached; checkFileIsNotAttached -> this checks that a specific file is not attached, others may be; checkFileIsAttached -> checks that a specific file is attached; checkFilesAreAttachedToWidget -> you can give multiple files as parameter